src/components/Views/SyncView/SyncView.js

```diff
@@ -114,78 +102,62 @@ class SyncView extends React.Component<
   };
 
   handleConnectionChange = (connectionInfo: Object) => {
-    if (connectionInfo.type === 'wifi') {
-      this.setState({ wifi: true });
-    } else {
-      this.setState({ wifi: false });
-    }
+    // TODO: re-announce once we aren't announcing every three seconds 
   };
 
   handleDevicePress = (item: Device) => {
     const { sync } = this.props;
-    console.log(item)
     const syncInProgress =
       item.syncStatus === 'replication-started' ||
-      item.syncStatus === 'replication-progress';
+      item.syncStatus === 'replication-progress' ||
+      item.syncStatus === 'replication-complete';
 
     if (!syncInProgress) {
       sync(item);
     }
   };
 
   renderItem = ({ item }: { item: Device }) => {
-    const { syncTarget } = this.props;
-
     return (
       <DeviceCell
         device={item}
         onPress={this.handleDevicePress}
-        selected={!!syncTarget && item.id === syncTarget}
       />
     );
   };
 
   handleBack = () => {
     const { unannounceSync, navigation } = this.props;
-    console.log('clicked back');
     this.stopSyncIntervals();
 
     navigation.goBack();
   };
 
   render() {
-    const { devices, navigation, syncTarget, sync } = this.props;
-    const { wifi } = this.state;
+    const { devices, navigation, sync } = this.props;
 
     let syncStopped = false;
-    const noDevices: boolean = devices.length === 0 || !wifi;
-
-    let progressText: string = noDevices
-      ? I18n.t('sync.none')
-      : I18n.t('sync.available');
-    const selectedDevice: ?Device = syncTarget
-      ? devices.find(device => device.id === syncTarget)
-      : undefined;
-
-    if (selectedDevice) {
-      console.log(selectedDevice.syncStatus)
-      switch (selectedDevice.syncStatus) {
-        case 'replication-started':
-          progressText = I18n.t('sync.initiated');
-          break;
-        case 'replication-progress':
-          progressText = I18n.t('sync.progress');
-          break;
-        case 'replication-error':
-          progressText = I18n.t('sync.stopped');
-          syncStopped = true;
-          break;
-        case 'replication-complete':
-          progressText = I18n.t('sync.completed');
-          break;
-        default:
-          progressText = I18n.t('sync.selected');
-      }
+    const noDevices: boolean = devices.length === 0;
+
+    let stuffHappening: ?Device = devices.find(device => device.syncStatus !== null)
```

Comment:
there is no 'selected device' screen yet, so this simply displays the correct indicator by translating the sync status to the appropriate progress text